src/main/java/com/rtb/projectmanagementtool/auth/AuthOps.java

```diff
@@ -27,18 +27,25 @@ public AuthOps(DatastoreService ds) {
     this.controller = new UserController(ds);
   }
 
-  public String getAuthID() {
```

Comment:
for `getAuthID` specifically, imo the previous version was more appropriate.  the concept of an AuthID is separate from the concept of a login cookie.  If you need to check for both, I would advice creating two separate methods (you already have one for checking the login cookie) and calling both one after the other